Tests/MeshGeoToolsLib/TestConstructAdditionalMeshesFromGeoObjects.cpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * @copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#include <ctime>
+#include <memory>
+#include <gtest/gtest.h>
+
+#include "GeoLib/GEOObjects.h"
+#include "GeoLib/Point.h"
+#include "MeshLib/Mesh.h"
+#include "MeshLib/MeshGenerators/MeshGenerator.h"
+
+TEST(ConstructAdditionalMeshesFromGeoObjects, PointMesh)
```

Comment:
Maybe an end-to-end test could cover the multiple nodes case; one failing when in default strict mode, another passing with multiple nodes allowed.